src/components/App.tsx

```diff
@@ -29,6 +30,20 @@ interface IAppProps extends RouteComponentProps<any> {
 @inject('mapStore', 'loginStore')
 @observer
 class App extends React.Component<IAppProps, IAppState> {
+    async componentWillMount() {
+        const pathName = navigator.getPathName();
+        if (!pathName.includes(SubSites.afterLogin)) {
+            const response = (await ApiClient
+                .postRequest(endpoints.EXISTING_SESSION, {}) as IAuthorizationResponse);
```

Comment:
Wouldn't it be better to set postRequest to take a optional parameter body? Then we could use it like:
postRequest(endpoints.EXISTING_SESSION)
postRequest(endPoints.LINK, link)

Reply:
Changing it to use getRequest instead